build/NuSpecs/ProjectReunionMetaPackage.nuspec

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>$ID$</id>
+    <version>0.0.0-SpecifyVersionOnCommandline</version>
+    <title>Microsoft Project Reunion Meta Package</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <description>Project Reunion Meta Package</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <tags>Windows Reunion ProjectReunion</tags>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <license type="file">license.txt</license>
+    <projectUrl>https://github.com/microsoft/projectreunion</projectUrl>
+    <iconUrl>https://aka.ms/winui_icon</iconUrl>
+    <dependencies> 
+      <!-- List all Project Reunion references here, please maintain in alphabetical order-->
+      <dependency id="Win2D.uwp" version="1.25.0"/>
```

Comment:
Is this data needed anywhere else? If so, do we generate those from this? Alternatively should those and this be generated from Dependencies.txt etc?

Reply:
I don't see the value of having another file that just puts the same data into the nuspec file.  Feel free to log an issue about this, though.  Not promising it will get done.